pull/github.go

```diff
@@ -60,6 +60,7 @@ func (loc Locator) IsComplete() bool {
 	case loc.Value.GetUser().GetLogin() == "":
 	case loc.Value.GetBase().GetRef() == "":
 	case loc.Value.GetBase().GetRepo().GetID() == 0:
+	case loc.Value.GetDraft() == false:
```

Comment:
I _think_ this means that non-draft PRs (e.g. most of them) will never be considered complete.